### DISCUSSION TITLE
Include the SHA

### DISCUSSION THREAD

**Discussion by vRune4** [GitTools/GitVersion#4045] [Q&A] [answered]
The InformationalVersion comes close to what I want: `  "InformationalVersion": "0.1.1+Branch.somebranch-workflow.Sha.9f414bc6ed448016936bfdfceda2a009ab2add63",`, but docker does not accept + in its tags.

Does gitversion offer a way to format the FullSemVer output for pull requests? I'd like to avoid having a separate formatting step in my workflow if possible.

I'm using 6.0RC1.

**Comment by asbjornu** [comment, marked as answer]
[As documented](https://gitversion.net/docs/reference/configuration#assembly-file-versioning-format):

> Expressions in curly braces reference one of the [variables](https://gitversion.net/docs/reference/variables) or a process-scoped environment variable (when prefixed with `env:`).

So you are able to configure some of the output variables of GitVersion to include basically anything you want through GitVersion's other output variables and/or environment variables provided by the execution environment.
